spec/factories/dataset.rb

```diff
@@ -67,6 +67,10 @@
       licence_custom '["Special case"]'
     end
 
+    trait :with_custom_licence_brackets_middle do
```

Comment:
Should we add a test which uses this trait?

Reply:
sure, forgot to add